docs/enso-protocol.md

```diff
@@ -0,0 +1,14 @@
+Enso protocol consists of pair server and client. They communicate by exchanging
+messages, following JSON-RPC 2.0.
+
+# Setup
+Follow the guidelines of [Enso repository](https://github.com/luna/enso) to setup the project
```

Comment:
Which exactly guidelines? I do not see "guidelines" at the provided link? Moreover, please make docs as easy to follow as possible. It should be very clear how to run this thing.